src/ncp/spinel.c

```diff
@@ -1051,918 +1051,1027 @@ spinel_ssize_t spinel_datatype_vpack(uint8_t *     data_ptr,
 
 // **** LCOV_EXCL_START ****
 
+const char *spinel_command_to_cstr(unsigned int command)
+{
+    const char *ret = "UNKNOWN";
+
+    switch (command)
+    {
+    case SPINEL_CMD_NOOP:
+        ret = "NOOP";
+        break;
+
+    case SPINEL_CMD_RESET:
+        ret = "RESET";
+        break;
+
+    case SPINEL_CMD_PROP_VALUE_GET:
+        ret = "PROP_VALUE_GET";
+        break;
+
+    case SPINEL_CMD_PROP_VALUE_SET:
+        ret = "PROP_VALUE_SET";
+        break;
+
+    case SPINEL_CMD_PROP_VALUE_INSERT:
+        ret = "PROP_VALUE_INSERT";
+        break;
+
+    case SPINEL_CMD_PROP_VALUE_REMOVE:
+        ret = "PROP_VALUE_REMOVE";
+        break;
+
+    case SPINEL_CMD_PROP_VALUE_IS:
+        ret = "PROP_VALUE_IS";
+        break;
+
+    case SPINEL_CMD_PROP_VALUE_INSERTED:
+        ret = "PROP_VALUE_INSERTED";
+        break;
+
+    case SPINEL_CMD_PROP_VALUE_REMOVED:
+        ret = "PROP_VALUE_REMOVED";
+        break;
+
+    case SPINEL_CMD_NET_SAVE:
+        ret = "NET_SAVE";
+        break;
+
+    case SPINEL_CMD_NET_CLEAR:
+        ret = "NET_CLEAR";
+        break;
+
+    case SPINEL_CMD_NET_RECALL:
+        ret = "NET_RECALL";
+        break;
+
+    case SPINEL_CMD_HBO_OFFLOAD:
+        ret = "HBO_OFFLOAD";
+        break;
+
+    case SPINEL_CMD_HBO_RECLAIM:
+        ret = "HBO_RECLAIM";
+        break;
+
+    case SPINEL_CMD_HBO_DROP:
+        ret = "HBO_DROP";
+        break;
+
+    case SPINEL_CMD_HBO_OFFLOADED:
+        ret = "HBO_OFFLOADED";
+        break;
+
+    case SPINEL_CMD_HBO_RECLAIMED:
+        ret = "HBO_RECLAIMED";
+        break;
+
+    case SPINEL_CMD_HBO_DROPED:
```

Comment:
Should this be `SPINEL_CMD_HBO_DROPPED`?

Reply:
Indeed! The enum definition itself has a typo. Will fix it.